pkg/kubelet/kubelet.go

```diff
@@ -1065,6 +1065,46 @@ func (kl *Kubelet) cleanupOrphanedVolumes(pods []*api.Pod, runningPods []*kubeco
 	return nil
 }
 
+// filterOutPodsExceedRuntimeTtl filters pods that have a container whose startAt time is more
+// than the specified RunningTtl for the pod.  It records an event that the pod has run longer
+// than the allocated time, and updates the pod status as failed.  By filtering the pod from
+// the result set, the Kubelet will kill the pod's containers as part of normal SyncPods workflow
```

Comment:
How this will be interactivate with restart policy -- RestartOnFailure and RestartAlways?